presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -131,7 +131,6 @@ public void testSchemaOperations()
 
     @Test
     public void testInformationSchemaTablesWithoutEqualityConstraint()
-            throws Exception
```

Comment:
Maybe we could remove such `throws` project wise?

Reply:
Please go ahead :)